function_app.py

```diff
@@ -36,39 +21,18 @@ def upload_file_to_blob(storage_account_name, container_name, file_path, blob_na
         blob_client.upload_blob(data, overwrite=True)
     return blob_client.url
 
-@app.route(route="ffmpeg_trrigger")
+@app.route(route="ffmpeg_trigger")
 def main(req: func.HttpRequest) -> func.HttpResponse:
     # Azure File Share connection details
     storage_account_name = os.getenv('AZURE_STORAGE_ACCOUNT_NAME')
     container_name = 'azurefiles'

```

Comment:
could we rename to ffmpegfiles or something like that to not confuse with Azure Files service?